config/locales/jobs.yml

```diff
@@ -3,7 +3,7 @@ en:
     heading: Welcome to Teaching Vacancies
     intro: >-
       Teaching Vacancies is a free job-listing service from the Department for Education.
-      You can search for a job at a school or Trust in England and set up job alerts.
+      You can search for a job at a school or trust in England, create an account and set up job alerts.
```

Comment:
What about saving jobs? That's my favourite bit 😂

Reply:
That is a good point actually, maybe we should just change 'create an account' to 'save jobs'?